docs/linux/external_fuzzing_usbip.md

```diff
@@ -0,0 +1,35 @@
+# **USB/IP Fuzzing for Linux Kernel**
+
+Syzkaller supports fuzzing the Linux kernel USB/IP subsystem externally. We can set up a virtual network and send USB/IP packets to the client kernel as they are being received from an external server.
+USB/IP fuzzing needs USB/IP configurations to be enabled. You can find the list in the configurations part.
+
+For the moment, USB/IP Fuzzing consists of fuzzing the client's side and can be explained in 2 main parts:
```

Comment:
Let's say: `Currently syzkaller only includes support for fuzzing the client side of USB/IP, which consists of 2 main parts`